CMakeLists.txt

```diff
@@ -555,6 +555,13 @@ install(
   PATTERN "CMakeLists.txt" EXCLUDE
 )
 
+# To override this, use '-DWL_INSTALL_BINDIR=<absolute path>' when configuring or change
+# 'CMAKE_INSTALL_PREFIX'.
+# Example: to have the bin installed to "/usr/games", just use '-DCMAKE_INSTALL_PREFIX=/usr'
+# and don't specify '-DWL_INSTALL_BINDIR' when configuring.
+# Note: 'CMAKE_INSTALL_PREFIX' defaults to /usr/local on UNIX and c:/Program Files/${PROJECT_NAME} on Windows
+set(WL_INSTALL_BINDIR "${CMAKE_INSTALL_PREFIX}/games" CACHE PATH "Destination of executable files (if installing)")
```

Comment:
> The daily PPA is failing now

Uh, oh. The value for WL_INSTALL_BINDIR here is set after including the src directory, so it is not picked up properly if not specified on the command line.